javascript/Components/SortListContainer/SortListItem.js

```diff
@@ -17,16 +26,27 @@ class SortListItem extends JSXComponent {
       </div>
     );
 
-    $(filedSelection).click(() => {
+    $(filedSelection).click((event) => {
       event.stopPropagation();
+      $(".btn-options").hide();
```

Comment:
make specific selectors to avoid issues when 2 tables are used.

Reply:
changes made